action.yml

```diff
@@ -0,0 +1,28 @@
+name: Rust Marker Linter
+description: GitHub Action to install and run Marker linting for Rust 🦀
+branding:
+  icon: edit-3
+  color: white
```

Comment:
Branding color and icon are used for GitHub Marketplace. You may change as you wish according to [the docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#brandingcolor) there is a selection of colours and Icons that you can preview

Reply:
I like the icon and color you picked out, so I'm in favor of keeping these :D